### PR TITLE
Clarify stroke dash array ordering

### PR DESCRIPTION
**Pull request by b-wils**

### CODE REVIEW

File: docs/specs/shapes.md [hunk 1]
```diff
@@ -473,6 +473,8 @@ come before them in [[stacking order]].
 
 {schema_string:shapes/stroke-dash/description}
 
+A stroke dash array consists of `n` dash entries, `[n-1,n]` gap entries and `[0-1]` offset entries. Dash and gap entries MUST all be in a continuous order and alternate between dash and gap, starting with dash. If there are an odd number of dashes + gaps, the sequence will repeat with dashes and gaps reversed. For example a sequence of `[4d, 8g, 16d]` MUST be rendered as `[4d, 8g, 16d, 4g, 8d, 16g]`.
+
```
Comment (mbasaglia) [inline docs/specs/shapes.md:477]: I think it needs to specify how the presence of the offset is determine. Presumably checking the type of the last item? (is offset always at the end?)

**Comment by mbasaglia** [comment]
Also, do we want this for 1.0.1 or 1.1?

**Comment by mbasaglia** [comment]
@cla-bot check

File: docs/specs/shapes.md (hunk 1 above)
Comment (b-wils) [reply docs/specs/shapes.md:477]: As spec is currently written, offset could be anywhere in the array. Bodymovin seems to always export this at the end and I know Skottie assumes this position as well. The stroke example has offset as the first element which lottie-web seems to handle.

I think restricting offset to always be the last item makes implementations simpler, though I'm not sure if other authoring tools export this in a different position.

**Comment by b-wils** [comment]
> Also, do we want this for 1.0.1 or 1.1?

My thought was 1.0.1 as the feature is already in the spec and this is clarifying the wording. It is a bit more substantial of an edit though, so could see a reason to push this to 1.1.

File: docs/specs/shapes.md (hunk 1 above)
Comment (b-wils) [reply docs/specs/shapes.md:481]: Updated this to specify offset MUST be at end. IMO it's better to be more restrictive to start an we can loosen if there is a need.

**Review by mbasaglia — APPROVED** [review]

**Review by fmalita — APPROVED** [review]

**Pull request merged**